### PR TITLE
MGMT-14787: Adding tests for events pagination

### PR DESCRIPTION
**Pull request by jgyselov**
https://issues.redhat.com/browse/MGMT-14787

### CODE REVIEW

File: libs/ui-lib-tests/cypress/views/pages/ClusterPage.ts [hunk 1]
```diff
@@ -0,0 +1,13 @@
+export default class ClusterPage {
+  static visit(options?: Partial<Cypress.VisitOptions>) {
+    return cy.visit(`/clusters/${Cypress.env('clusterId')}`, options);
+  }
```
Comment (jkilzi) [inline libs/ui-lib-tests/cypress/views/pages/ClusterPage.ts:4]:
```suggestion
  static visit(clusterId = Cypress.env('clusterId'), options?: Partial<Cypress.VisitOptions>) {
    return cy.visit(`/clusters/${clusterId}`, options);
  }
```

File: libs/ui-lib-tests/cypress/views/modals/eventsModal.ts [hunk 2]
```diff
@@ -0,0 +1,93 @@
+import ClusterPage from '../pages/ClusterPage';
+
+export default class EventsModal {
```
[the remark this answers is not in the document]
Comment (jkilzi) [reply libs/ui-lib-tests/cypress/views/modals/eventsModal.ts:3]: Sorry, after thinking about this again we might hit a cyclic dependency problem due to having the EventsModal importing the ClusterPage.

The EventsModal (and all of its parts) appear not only in the ClusterPage, so imagine you would like to add a test for the installation page. Given the way this "page-object" is written the installation page will transitively import the cluster page (now what have both to do with each other if the test we are writing is only concerned about the events modal)

[the remark this answers is not in the document]
Comment (jkilzi) [reply libs/ui-lib-tests/cypress/views/modals/eventsModal.ts:3]: Let's sync on this I think I have a better solution

**Review by jkilzi — CHANGES_REQUESTED** [review]

File: libs/ui-lib-tests/cypress/fixtures/cluster/events.ts [hunk 3]
```diff
@@ -0,0 +1,344 @@
+import { hostIds as fakeHostIds } from '../hosts';
+import { fakeClusterId, fakeClusterInfraEnvId } from './base-cluster';
+
+const clusterEvents = [
```
[the remark this answers is not in the document]
Comment (ammont82) [reply libs/ui-lib-tests/cypress/fixtures/cluster/events.ts:4]: I agree with @jkilzi . Maybe you can implement a function to make events.

**Review by ammont82 — DISMISSED** [review]

**Review by ammont82 — APPROVED** [review]

**Review by jkilzi — APPROVED** [review]

**Pull request merged**